### ISSUE TITLE
Installation Issues on OSX, with npm 3.7.2 and node v5.1.0

### ISSUE THREAD

**Issue by seahyc** [yjwong/node-podofo#2]
Hey yjwong!

I'm facing an issue with npm install with the abovementioned configuration, did I miss out any configuration before this?

```
~ npm install podofo

> podofo@0.2.0 install /Users/syc/Code/glints/api/node_modules/podofo
> node-pre-gyp install --fallback-to-build

  CXX(target) Release/obj.target/podofo/PdfContentsTokenizer.o
In file included from ../PdfContentsTokenizer.cc:2:
../PdfPage.h:7:10: fatal error: 'podofo.h' file not found
#include "podofo.h"
         ^
1 error generated.
make: *** [Release/obj.target/podofo/PdfContentsTokenizer.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/syc/.nvm/versions/node/v5.1.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 15.3.0
gyp ERR! command "/Users/syc/.nvm/versions/node/v5.1.0/bin/node" "/Users/syc/.nvm/versions/node/v5.1.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding/podofo.node" "--module_name=podofo" "--module_path=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding"
gyp ERR! cwd /Users/syc/Code/glints/api/node_modules/podofo
gyp ERR! node -v v5.1.0
gyp ERR! node-gyp -v v3.2.1
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/Users/syc/.nvm/versions/node/v5.1.0/bin/node /Users/syc/.nvm/versions/node/v5.1.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding/podofo.node --module_name=podofo --module_path=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/syc/Code/glints/api/node_modules/podofo/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:818:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
node-pre-gyp ERR! System Darwin 15.3.0
node-pre-gyp ERR! command "/Users/syc/.nvm/versions/node/v5.1.0/bin/node" "/Users/syc/Code/glints/api/node_modules/podofo/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/syc/Code/glints/api/node_modules/podofo
node-pre-gyp ERR! node -v v5.1.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.12
node-pre-gyp ERR! not ok
Failed to execute '/Users/syc/.nvm/versions/node/v5.1.0/bin/node /Users/syc/.nvm/versions/node/v5.1.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding/podofo.node --module_name=podofo --module_path=/Users/syc/Code/glints/api/node_modules/podofo/lib/binding' (1)
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/syc/.nvm/versions/node/v5.1.0/bin/node" "/Users/syc/.nvm/versions/node/v5.1.0/bin/npm" "install" "podofo"
npm ERR! node v5.1.0
npm ERR! npm  v3.7.2
npm ERR! code ELIFECYCLE

npm ERR! podofo@0.2.0 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the podofo@0.2.0 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the podofo package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs podofo
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls podofo
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/syc/Code/glints/api/npm-debug.log
```

**Comment by seahyc** [comment]
Resolved

**Comment by yjwong** [comment]
Resolved in 172b643f and also pushed OSX binaries to S3. Thanks!
